modules/cli/pages/index.adoc

```diff
@@ -51,12 +49,257 @@ $ mv skupper $HOME/bin
 +
 [subs=attributes+]
 ----
-$ skupper --version
-skupper version {skupper-version}
+$ skupper version
+client version {skupper-version}
+----
+// end::installing-cli[]
+
+// Type: procedure
+[id="creating-using-cli"] 
+== Creating a site using the CLI
+
+A {service-network} consists of {skupper-name} sites.
+This section describes how to create a site using the default settings.
+
+.Prerequisites
+
+* The `skupper` CLI is installed.
+* You are logged into the cluster.
+* The services you want to expose on the {service-network} are in the active namespace.
+
+
+.Procedure
+
+. Create a default site:
++
+----
+$ skupper init
+----
+
+. Check the site:
++
 ----
+$ skupper status
+
+Skupper is enabled for namespace "west" in interior mode. It is not connected to any other sites.
+----
+
+The default settings include:
 
+* console - The {skupper-name} console is provisioned with a single user.
+The password for the `admin` user is stored in the `skupper-console-users` secret. 
+For more information on the console, see link:{using-console-link}[Using the {skupper-name} console].
+
+* site name - The site name defaults to the namespace name, for example, `west`.
 
 // Type: procedure
 [id="creating-using-cli"] 
-== Creating a Skupper site using the CLI
+== Customizing a site using the CLI
+
+If you require a custom configuration, you might need to recreate the site.
+
+.Procedure
+
+
+. Create a site without a console:
++
+----
+$ skupper delete
+$ skupper init --enable-console false
+----
++
+NOTE: If you have linked to sites, you need to recreate those links after reinitializing the site.
+
+. Configure the {skupper-name} console protection.
+There are a number of `skupper` options regarding authentication for the console:
++
+--
+`--console-auth <authentication-mode>`:: Set the authentication mode for the console:
+
+* `openshift` - Use OpenShift authentication, so that users who have permission to log into OpenShift and view the Project (namespace) can view the console.
+* `internal` -  Use {skupper-name} authentication, see the `console-user` and `console-password` options.
+* `unsecured` - No authentication, anyone with the URL can view the console.
+
+
+`--console-user <username>`:: Username for the console user when authentication mode is set to `internal`.
```

Comment:
Maybe worth saying that the default is "admin" for `--console-user` and
a random password is generated if `--console-password` is not provided?